### ISSUE TITLE
Add descriptions to settings options

### ISSUE THREAD

**Issue by nstgeorge** [nstgeorge/VEKTA#71]
### Short Description

Provide some additional detail about what each setting does to the user.

### More Details

This will probably involve a slight redesign to the settings page, since there's not really a great place to put a description. I think moving the menu to the left and providing the details (and possibly a slightly longer name) for the selected option.

**Comment by rvanasa** [comment]
Yes, and this is a perfect use case for the `SideLayoutMenuHandle` and `InfoAware` pattern.
